contrib/!lang/go/packages.el

```diff
@@ -11,6 +11,9 @@
     (add-hook 'go-mode-hook 'flycheck-mode))
 
 (defun go/init-go-mode()
+  (when (memq window-system '(mac ns x))
```

Comment:
Following the pattern used in dc5767ae0c0096f5998efa3a8a931dc3e7a92582. `exec-path-from-shell` is only initialized on those systems.